apps/style/applab/style.scss

```diff
@@ -500,3 +502,12 @@ button.arrow:focus {
   background-size: 16px 16px;
   background-position: 2px 0px;
 }
+
+.design-mode-hidden {
```

Comment:
super optional CSS trick -- if you want to emphasize the overriding behavior, these can be combined as:

.design-mode-hidden {
  visibility: hidden;
  divApplabDesignMode & {
    visibility: visible;
    opacity: 0.3;
  }
}